src/neo/SmartContract/Native/Tokens/GasToken.cs

```diff
@@ -54,6 +54,16 @@ protected override bool OnPersist(ApplicationEngine engine)
             return true;
         }
 
+        [ContractMethod(0, ContractParameterType.Boolean, ParameterTypes = new[] { ContractParameterType.Hash160, ContractParameterType.Integer }, ParameterNames = new[] { "account", "amount" })]
+        private StackItem OnRecycleRewardGas(ApplicationEngine engine, VMArray args)
+        {
+            if (engine.Trigger != TriggerType.System) return false;
+            UInt160 acount = new UInt160(args[0].GetSpan());
+            BigInteger amount = args[1].GetBigInteger();
+            Mint(engine, acount, amount);
```

Comment:
It's hard to trace this, the user cant difference between a Transfer and a recycle, the notification it's the same.

Reply:
Here I refer to the use of the distribution of NetworkFee's `Mint` in OnPersist. If the notification in the Mint method needs to be distinguished, we need to distinguish the general transfer, recycle, 
 distribution network fees, distribution system fees, and system initialization fees. Is it necessary to distinguish?

Reply:
My ideal explorer should be able to show how many gas was recycle, I think that we should be able to distinguish this, maybe we need to modify NEP5, in order to add a Tag in `Transfer` event.

https://github.com/neo-project/proposals/blob/master/nep-5.mediawiki#transfer-2

Reply:
I think we can parameterize the content of notification in `Mint`, pls check my commit.